Manifests/ManagedPreferencesApplications/com.google.Chrome.plist

```diff
@@ -15,7 +15,7 @@
 	<key>pfm_interaction</key>
 	<string>combined</string>
 	<key>pfm_last_modified</key>
-	<date>2020-02-06T10:00:00Z</date>
+	<date>2020-04-08T20:22:48Z</date>
```

Comment:
Recent Chrome PR has a newer last_modified date, so this would need to be updated.

Reply:
Hey @apizz that's implemented in the PR on a commit more recent than the one you cited. Check out b31199e.